client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/test/site-content.tsx

```diff
@@ -3,6 +3,7 @@
  */
 
 import { render } from '@testing-library/react';
+import { QueryClient, QueryClientProvider } from 'react-query';
```

Comment:
Couldn't we wrap a component higher in the component chain to avoid wrapping many components?

Reply:
Thanks for the review, I have addressed your comments.